src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -103,12 +100,34 @@ public ArmSubsystem(int armId, String armCanbus, int channel1)
   //Looking at the left of the robot, counterclockwise arm spin is positive
   public void spinArm(double speed)
   {
-    if(speed < -15) { 
-      speed = -15;
+    if(speed < -Arm.MAX_VOLT) { 
+      speed = -Arm.MAX_VOLT;
     }
-    else if (speed > 15) { 
-      speed = 15;
+    else if (speed > Arm.MAX_VOLT) { 
+      speed = Arm.MAX_VOLT;
+    }
+
+    if(encoderGetAngle() >= 300) {
+      if (speed > 1) {
+        speed = 1;
+      }
     }
+
+    if(encoderGetAngle() >= 332) {
+      speed = -1;
+    }
+
+    if(isEmergencyStop()) {
+      speed = 0;
+    }
+
+    if(booster)
```

Comment:
I would do the booster check earlier, so the sanity checks around encoder angle will win.

Reply:
Also should the booster work in both directions? here, it's only the positive value, even if the arm should be moving the other direction